ansible/roles/prometheus-server/files/prometheus.yml

```diff
@@ -0,0 +1,31 @@
+# example global config
```

Comment:
Looking at this I think we should take out the commented out lines as they're obscuring the config. We have 7 lines of content in a 31 line file